packages/playground/src/utils/filter_nodes.ts

```diff
@@ -33,3 +36,85 @@ export async function getFilteredNodes(grid: GridClient, options: NodeFilters):
   const nodes = await grid.capacity.filterNodes(filters);
   return nodes;
 }
+
+// Node filters used in #Explorer, #Dedicated Nodes...
+
+// Input attrs
+export type NodeInputFilterType = {
+  label: string;
+  placeholder: string;
+  value?: string | undefined;
```

Comment:
Value could be string or number.

Reply:
The HTMLInputElement returns the value as string even if the value is number